### PR TITLE
Fix/rsvp acceptance testing

### PR DESCRIPTION
**Pull request by juanfra**
🎥 https://www.loom.com/share/4a182954b6a54eebaf2dde1f8c6364a7

**WIP:** Been playing a bit with acceptance testing for the new RSVP block.

### CODE REVIEW

File: tests/_support/Helper/TribeDb.php [hunk 1]
```diff
@@ -0,0 +1,309 @@
+<?php
```
Comment (sc0ttkclark) [inline tests/_support/Helper/TribeDb.php:1]: Can't add / call this from common?

Comment (juanfra) [reply tests/_support/Helper/TribeDb.php]: https://github.com/moderntribe/tribe-common/pull/1522/files

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Needs conflict resolution

**Comment by juanfra** [comment]
Hi @sc0ttkclark! Thanks for the review. I don't see any conflicts. Although this PR needs this code that was merged into G21.01 for common 👉 https://github.com/the-events-calendar/tribe-common/pull/1522

**Comment by sc0ttkclark** [comment]
Updated common dependencies

**Pull request merged**